Automattic/vip-go-mu-plugins
# Bring in Elasticpress

## Description
Fixes Term deletion in Elaticpress

Brings in:
* https://github.com/Automattic/ElasticPress/pull/117

## Changelog Description

### Plugin Updated: Enterprise Search

This change makes sure a term deletion is correctly propagated to ElasticSearch if the terms feature is enabled.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test


1) `dev-env exec -- wp elasticpress activate-feature terms`
2)  `dev-env exec -- wp vip-search index --setup`
3) Create some hierarchy of categories
4) Delete middle one `dev-env exec -- wp term delete category <ID of term that has children>
5) No errors
5) Term is deleted
5) Check that index is correctly showing the children of deleted term pointing to the parent of deleted term as their parent now
